### ISSUE TITLE
Tenon.io accessibility findings

### ISSUE THREAD

**Issue by gregdorward** [redbadger/website-honestly#469]
**Description of the issue:**

The following issues were found when running an accessibility test against Website Honestly

- [x] Social media links in the footer are missing link text throughout the site (fixing this would improve our accessibility score by quite a bit)

- [x] Remove duplicate ids, especially if they identify a control. 'Symbols' and 'arrow' are being used repeatedly. This looks to be especially bad on the 'About us' page

- [x] Some images are still missing alt text, e.g. img role="presentation class="styles__leanImage___7Cp9U" src="/assets-honestly/lean-2WF10.png">'

- [x] Duplicate alt-text on the Retailer case study page, img src="/assets-honestly/header-3SKLJ.jpg" alt="shopping cart" class="style__header__image___1GrxS"

- [x] Empty buttons on the About Us page (I think in the History Slice)

- [x] Alt text for some images used in social media may be a bit too long (need to be concise, yet informative

- [x] Same alt text on images with different src attributes - these should be distinct

- [x] Alt text should never consist only of numerics, e.g. '2012' on the 'About us' page

- [x] On the 'About us' page, the link text in this element is too uninformative a rel="noopener noreferrer" target="_blank" href="/services/technology">here</a

~- [ ] A lot of missing alt text and missing text in links on the blog pages. Not entirely sure what we can do about this, given the links and images are hosted in Squarespace~ Should be taken care of in a separate blog ticket

**Comment by gregdorward** [comment]
In order to run another Tenon check, we're going to have to wait until this one is in Staging. When running this branch locally we can still use other open source tools, such as Chrome's Web Developer toolbar > Validate Accessibility
